cluster/gce/trusty/node.yaml

```diff
@@ -133,12 +131,15 @@ Content-Disposition: attachment; filename="kube-install-additional-packages.conf
 
 description "Install additional packages used by kubernetes"
 
-start on started docker
+start on stopped kube-install-packages
 
 script
```

Comment:
I just verified using ubuntu trusty: "set -o pipefail" does not work but the other two work. I will add these two commands in node.yaml